### PR TITLE
Add Podtrics.com to PodcastIndex Apps Page

### PR DESCRIPTION
**Pull request by mbeinder**
This PR adds Podtrics.com to the PodcastIndex.org/apps page.

Podtrics is a podcast hosting platform focused on keeping creators creative.

### CODE REVIEW

**Comment by mbeinder** [comment]
I see that the CI Test failed, but it's from a section of the data.json file that I did not touch nor is related to Podtrics. Please advise on how I should proceed. @daveajones

**Comment by mbeinder** [comment]
@ericpp Okay I think we should be good now. I had to rebase from upstream master, and do some conflict resolution. Please let me know if you need me to do anything else.

**Comment by ericpp** [comment]
@mbeinder It looks like you have it apps.json twice now. BTW, I'm not actually a maintainer here, but I was looking through the PRs and was curious how you got the CI error.

**Comment by mbeinder** [comment]
@ericpp thanks for pointing that out! And thanks for clarifying! Hopefully @daveajones can get this approved and merged soon.

File: server/data/apps.json [hunk 1]
```diff
@@ -1930,5 +1960,5 @@
         "elementURL": "https://bitbadger.solutions/open-source/myweblog/rss.html#podcast-feeds"
       }
     ]
-  }
+  }
```
Comment (ericpp) [inline server/data/apps.json:1963]: I would remove the extra space too
```suggestion
  } 
```

**Comment by daveajones** [comment]
Sorry for the late merge!

**Comment by mbeinder** [comment]
All good! Thank you!!

**Pull request merged**